group/init/README.md

```diff
@@ -79,5 +61,40 @@ var plain_text = .... stringify + buffer
 
 var ciphertext = envelope(plain_text, feed_id, prev_msg_id, msg_key, [ group_key, own_key ])
 
-ciphertext ---> string + .box2
+// what you publish (enveloped plaintext)
+// ciphertext = "SDSDsadlksajda432wdfsdlkfja=.box2"
+```
+
+## Epoch init example
+
+```js
+const groupRoot = 'ssb:message/classic/Zz-Inkte70Qz1UVKUHIhOgo16Oj_n37PfgmIzLDBgZw=.sha256'
+const groupId = 'ssb:identity/group/g_JTmMEjG4JP2aQAO0LM8tIoRtNkTq07Se6h1qwnQKb='
+
+var plainText = {
+  type: 'group/init'
+  version: 'v2',
+  groupKey: group_key.toString('base64'),
+  tangles: {
+    group: {
+      root: groupRoot,
+      previous: [lastMessageOnPreviousEpoch]
+    },
+    epoch: {
+      root: groupRoot,
+      previous: [groupRoot]
```

Comment:
```suggestion
      previous: [lastGroupInitOnPreviousEpuch]
```

Reply:
:fire: I feel like this change is really important, because people fuck up `previous` ALL THE TIME

Reply:
done